suite-native/module-device-settings/src/screens/DeviceNameScreen.tsx

```diff
@@ -0,0 +1,51 @@
+import { Text, TitleHeader, VStack } from '@suite-native/atoms';
+import { Form, FormSubmitButton, TextInputField } from '@suite-native/forms';
+import { Translation } from '@suite-native/intl';
+import { Screen, ScreenHeader } from '@suite-native/navigation';
+
+import { MAX_LENGTH, useChangeDeviceName } from '../hooks/useChangeDeviceName';
+
+export const DeviceNameScreen = () => {
+    const { form, device, hintMessage, deviceNameValue, onSubmit, isSubmittable } =
+        useChangeDeviceName();
+
+    return (
+        <Screen header={<ScreenHeader closeActionType="close" />}>
+            <Form form={form}>
+                <VStack marginTop="sp32" spacing="sp32" style={{ marginBottom: 'auto' }}>
```

Comment:
```suggestion
                <VStack marginTop="sp32" spacing="sp32" justifyContent="space-between">
```

This should work, since you always render at least the wrapper for the submit button.

Reply:
![Screenshot 2025-06-24 at 15 05 49](https://github.com/user-attachments/assets/86ae93ea-80a1-4fb6-be48-da24205db5cb) This unfortunately won’t work, as the screen container is a `ScrollView`. It would need a concrete height.